astropy/config/paths.py

```diff
@@ -99,25 +109,25 @@ def get_config_dir(create=True):
     # If using set_temp_config, that overrides all
     if set_temp_config._temp_path is not None:
         xch = set_temp_config._temp_path
-        config_path = os.path.join(xch, 'astropy')
-        if not os.path.exists(config_path):
+        config_path = os.path.join(xch, "." + rootname)
```

Comment:
I don't understand the prepending of "." here. Was it added automatically before?

Reply:
It wasn't and it lead to a discrepancy between the path when testing and in normal use, this change makes the test path the same as the real one.